.github/workflows/maven.yml

```diff
@@ -13,17 +13,18 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [1.8, 11]
+        java: [8, 11]
         os: [ubuntu-latest, windows-latest, macos-latest]
     runs-on: ${{ matrix.os }}
     env:
       maven_commands: install # default is install
     steps:
       - uses: actions/checkout@v2
       - name: Set up JDK ${{ matrix.java }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
+          distribution: 'zulu'
       - name: Cache Maven packages
```

Comment:
Should this be replaced with `cache: 'maven'` in the `setup-java` step?

Reply:
yes. I forgot to do that in that repo.
Well spotted